pages/k8s/1.18/components.md

```diff
@@ -366,44 +355,27 @@ These are the container images used by this release:
 -   k8scsi/csi-resizer:v0.3.0
 -   k8scsi/csi-snapshotter:v1.0.2
 -   k8scsi/csi-snapshotter:v1.2.2
--   kubernetes-ingress-controller/nginx-ingress-controller-amd64:0.30.0
--   kubernetes-ingress-controller/nginx-ingress-controller-arm64:0.30.0
--   kubernetes-ingress-controller/nginx-ingress-controller-s390x:0.20.0
--   kubernetesui/dashboard-amd64:v2.0.0-rc5
--   kubernetesui/dashboard-arm64:v2.0.0-rc5
--   kubernetesui/dashboard-s390x:v2.0.0-rc5
--   kubernetesui/metrics-scraper:v1.0.3
+-   metrics-scraper:v1.0.3
 -   metrics-server-amd64:v0.3.6
--   metrics-server-arm64:v0.3.6
--   metrics-server-s390x:v0.3.6
--   nvidia/k8s-device-plugin:1.0.0-beta5
-
+-   microbot-amd64:latest
+-   microbot-arm64:latest
+-   nginx-ingress-controller-amd64:0.30.0
+-   nginx-ingress-controller-arm64:0.30.0
+-   nginx-ingress-controller-s390x:0.20.0
```

Comment:
Why remove the `kubernetes-ingress-controller/nginx-ingress-controller-x` prefix?